satchless/payment/__init__.py

```diff
@@ -4,6 +4,14 @@ def __init__(self, error_message):
         self.error_message = error_message
 
 
+class PaymentsGatewayReceiptFormError(PaymentFailure):
+    def __init__(self, form):
+        error_message = ', '.join(
```

Comment:
Fixed in 8688eb0075f069756623fb12481ae300c591bccf.  Good suggestions - thank you kindly.